csrc/quantization/compressed_tensors/int8_quant_kernels.cu

```diff
@@ -282,6 +282,7 @@ void static_scaled_int8_quant(torch::Tensor& out,          // [..., hidden_size]
   TORCH_CHECK(input.is_contiguous());
   TORCH_CHECK(out.is_contiguous());
   TORCH_CHECK(scale.numel() == 1);
+  TORCH_CHECK(scale.item<float>() != 0.f, "Scale value must be non-zero.");
```

Comment:
this is only a check right? where is the actual change?

Reply:
Thanks for the reminder! My bad, maybe some issue with git, now fixed